R/prepare-admin-panel.R

```diff
@@ -927,7 +933,7 @@ prepare_admin_panel_components <- function(
         content = dplyr::case_when(
           type %in% c("login", "logout") ~ type,
           type == "input" ~ sprintf("Input: %s <br /> Value: %s", id, value),
-          type == "navigation" ~ sprintf("Navigated: %s", id),
+          type == "navigation" ~ sprintf("Navigated (%s): %s", id, value),
```

Comment:
Improved details (showing id of input tracking navigation and value)

Reply:
A very small but nice improvement. Nice!